front/src/components/user-menu/user-menu.component.css

```diff
@@ -0,0 +1,51 @@
+.v-avatar {
+  border: 4px solid #195292;
+}
+
+.v-btn {
+  border: 2px solid #195292;
+  text-transform: none;
+}
+
+  .v-btn:hover {
+    color: white;
+    background-color: #195292 !important;
```

Comment:
it is bad signal about !important, you should find ways to make this thing without important. might, it is more specific selector with custom specified class, such as mmt-btn, mmt or like this

Reply:
removed all !important from css